package/gluon-ebtables-source-filter/check_site.lua

```diff
@@ -0,0 +1,2 @@
+need_string_match('prefix4', '^%d+.%d+.%d+.%d+/%d+$', false)
+need_string_array_match('additional_prefix6', '^[%x:]+/%d+$', false)
```

Comment:
I think this should be called "additional_prefixes6", and we should have the same for IPv4 for consistency.